ml-search-web/src/app/store/actions.ts

```diff
@@ -16,4 +16,9 @@ export class CompleteSearchAction implements Action {
   constructor(public searchData: SearchData) {}
 }
 
+export class ChangePage implements Action {
```

Comment:
do we really need this? On change page we will be searching again with new pageNumber.

Reply:
ChangePage removed as we dont require this